docs/master/api-reference/directives.md

```diff
@@ -2877,13 +2891,14 @@ type Person {
 
 type Query {
     people(
-        where: WhereConstraints @whereConstraints(columns: ["age", "type", "haircolour", "height"])
+        where: _ @whereConstraints(columns: ["age", "type", "haircolour", "height"])
```

Comment:
@CPSibo i removed the requirement to set the correct type in the schema and just tell the user to put `_` as a placeholder, do you think that is more clear then what we had before. You mentioned this part was confusing you.